OpenTokSDKVersion.rb

```diff
@@ -1 +1,2 @@
-OpenTokSDKVersion = '2.20.1'
+OpenTokSDKVersion = '2.23.1'
+MinSdkVersion = '12.0'
```

Comment:
Maybe this could have a more explicit name (as it refers to iOS sdk versio, right?). But not that important...

Reply:
Yes Yes. I agree